python/hail/dataset.py

```diff
@@ -3274,10 +3274,9 @@ def mendel_errors(self, pedigree):
 
     @handle_py4j
     @record_method
-    @typecheck_method(max_shift=integral)
-    def min_rep(self, max_shift=100):
-        """
-        Gives minimal, left-aligned representation of alleles. Note that this can change the variant position.
+    @typecheck_method(left_aligned=bool)
+    def min_rep(self, left_aligned=False):
+        """Gives minimal, left-aligned representation of alleles. Note that this can change the variant position.
```

Comment:
Should clarify this -- doesn't actually left-align, just truncates identical bases from the left and right and can move the position forward.

Reply:
Can I leave this for a separate PR?  Once filter_alleles is in, too, I'll update all the allele-based comments to have accurate and consistent language.

Reply:
Sure